app.js

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable linebreak-style */
+/* eslint-disable max-classes-per-file */
+
+// Book Class: Represents a book
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+// Store Class: Handles Storage
+class Store {
+  // Check if there is a local storage for books
+  static #checkStorage() {
+    return localStorage.getItem('books');
+  }
+
+  // Get all the books Infos from the local storage
+  static getBooks() {
+    let books = [];
+    if (Store.#checkStorage()) {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+
+    return books;
+  }
+
+  // Save a book in local storage
+  static saveBook(book) {
+    const books = Store.getBooks();
+
+    books.push(book);
+
+    localStorage.setItem('books', JSON.stringify(books));
+  }
+
+  // Delete a book in local storage
+  static unSaveBook(title) {
+    const books = Store.getBooks();
+
+    books.forEach((book, index) => {
+      if (book.title === title) {
+        books.splice(index, 1);
+      }
+    });
```

Comment:
- [OPTIONAL] Great job so far 👍🏻 but I would suggest using array methods that don't mutate the array, e.g. `concat`, `filter`, `slice` instead of `push`, `pop`, `splice` and so on.For more information, visit this link about [How to Write Remarkably Clean JavaScript Code
](https://www.sitepoint.com/immutable-array-methods-javascript/)
eg:

```javascript
  addBook(title, author) {
    id = generateID(); 
    this.books = this.books.concat({ title, author, author });
    return this.books; 
  }
```

Reply:
Thank you @BenMukebo for sharing that information 👏👏👏, I'll consider doing that from now on.